src/apps/loan-list/materials/selectable-material.tsx

```diff
@@ -67,27 +67,26 @@ const SelectableMaterial: FC<SelectableMaterialProps & MaterialProps> = ({
           </p>
         </div>
         <div className="list-materials__status">
-          {/* todo this will be changed, everything with these statusses will be revised */}
           {renewalStatusList && (
-            <span className="text-small-caption">
-              {renewalStatusList.includes("deniedMaxRenewalsReached") && (
-                <>{t("LoanListDeniedMaxRenewalsReachedText")}</>
-              )}
-              {(renewalStatusList.includes("deniedOtherReason") ||
-                renewalStatusList.includes("deniedReserved")) && (
-                <> {t("LoanListDeniedOtherReasonText")}</>
-              )}
-              {/* todo "Lånet er fornyet i dag" -> this information is lacking in fbs */}
+            <>
+              {renewalStatusList.map((text) => (
+                <span className="text-small-caption">
+                  {getStatusText(text, t)}
+                </span>
+              ))}
               {loanType === "interLibraryLoan" && (
-                <>{t("loanListDeniedInterLibraryLoanText")}</>
+                <span className="text-small-caption">
+                  {t("groupModalRenewLoanDeniedInterLibraryLoanText")}
+                </span>
               )}
-            </span>
+            </>
           )}
           {dueDate && (
             <StatusBadge
               dueDate={dueDate}
-              neutralText={`${t("loanListToBeDeliveredMaterialText")} 
-            ${formatDate(dueDate)}`}
+              neutralText={t("groupModalDueDateMaterialText", {
+                placeholders: { "@count": formatDate(dueDate) }
```

Comment:
`@count`pointing at a date? Is it on purpose or is it a mistake?

Reply:
mistake, ill fix